Makefile

```diff
@@ -33,6 +35,7 @@ override CFLAGS += -Wextra -Wshadow -Wjump-misses-init -Wundef
 # Remove missing-field-initializers because of GCC bug
 override CFLAGS += -Wno-missing-field-initializers
 
+override CFLAGS += $(EXTRA_CFLAGS)
```

Comment:
LTO should be moved into the `ifdef DEBUG` section above, otherwise it would make debugging difficult.

Reply:
ok, done